src/components/terminal/Terminals.tsx

```diff
@@ -136,8 +115,19 @@ const Terminals: React.FunctionComponent<ITerminalsProps> = ({
           >
             {terminals.map((terminal, index) => {
               return (
-                <IonSegmentButton key={index} value={`term_${index}`}>
-                  <IonLabel>{terminal.name}</IonLabel>
+                <IonSegmentButton key={index} value={`term_${index}`} layout="icon-end">
+                  <IonButton
+                    fill="clear"
+                    className="terminal-tab-close-button"
+                    onClick={(e) => {
+                      e.stopPropagation();
+                      e.preventDefault();
+                      removeTerminal(index);
+                    }}
+                  >
+                    <IonIcon slot="icon-only" icon={close} className="terminal-tab-close-button-color" />
+                  </IonButton>
+                  <IonLabel className="terminal-tab-label">{terminal.name} iajsdoi ajsoidj aoisjd ioajs doij</IonLabel>
```

Comment:
oops? dont think `iajsdoi ajsoidj aoisjd ioajs doij` is ment to be here as its displayed in the tab name too

Reply:
Oh yes, this was just for testing long names. Thank you 🙂